scripts/boot.upstart.conf

```diff
@@ -0,0 +1,7 @@
+description "boot config"
+start on (local-filesystems and net-device-up IFACE!=lo)
+respawn
+env DISPLAY=:0.0
+script 
+	cd /var/goddard/agent && chmod a+x scripts/boot.sh && ./scripts/boot.sh | tee /var/log/goddard.boot.log tty1
+end script
```

Comment:
No we split them up. One does provisioning of the node itself, which then just makes sure the node is setup and happy. While the boot.sh is then started from that initial setup which will keep running and trying to provision. See new commit just updated so the only way it's started is by the init.sh script.